pyuvsim/simsetup.py

```diff
@@ -358,6 +358,8 @@ def initialize_catalog_from_params(obs_params, input_uv=None):
 
 def _construct_beam_list(beam_ids, telconfig):
     beam_list = BeamList([])
+    if 'spline_interp_opts' in telconfig.keys():
```

Comment:
should we add anything to the docs about this option?

Reply:
I've added this and `freq_interp_kind` to the documentation on parameter files.